doc/getting-started/apl-nuc.rst

```diff
@@ -177,7 +177,8 @@ partition. Follow these steps:
       whether it should be disabled. There are three forms for this parameter:
 
       #. ``uart=disabled``: this disables the serial port completely
-      #. ``uart=mmio@<MMIO address>``: this sets the serial port MMIO address
+      #. ``uart=bdf@<BDF value>``:  this sets the PCI serial port based on its BDF.
+      For example, use ``bdf@0:18.2`` for a BDF of 0:18.2 ttyS2.
```

Comment:
You need this line to be aligned with the content of the bullet list, i.e. align its start with the first quote from the line above.

Reply:
OK, thanks! is it like this one? 
#. ``uart=bdf@<BDF value>``:  this sets the PCI serial port based on its BDF.  
      For example, use ``bdf@0:18.2`` for a BDF of 0:18.2 ttyS2.